### ISSUE TITLE
I am getting npm error on one of my apps due to node version

### ISSUE THREAD

**Issue by s7dhansh** [zodern/mup-aws-beanstalk#71]
```
gyp ERR! cwd /tmp/deployment/application/programs/server/npm/node_modules/meteor/montiapm_agent-binary-deps/node_modules/v8-profiler
gyp ERR! node -v v10.14.1
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok. 
```

I am unable to figure out why? It is happening on only one app of the 4 I have deployed so far, all with the latest meteor 1.8.0.1. Somehow nvm is using node 10 instead of node 8. 
Can someone kindly help?

**Comment by bakuur** [comment]
I'm still having issues with this which is halting the entire deployment process

my meteor version is 1.8.0.2 and i keep getting all sorts of error with npm and node.js while deploying instances on beanstalk

any assistance would be appreciated

**Comment by zodern** [comment]
This should be fixed in 0.7 - #142
